packages/tradershub/src/features/cfd/components/OtherCFDPlatformsList/OtherCFDPlatformsList.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC, useEffect } from 'react';
+import { useAuthorize, useDxtradeAccountsList, useInvalidateQuery } from '@deriv/api';
+import { Text } from '@deriv/quill-design';
+import { AddedDxtradeAccountsList, AvailableDxtradeAccountsList } from '../../flows/OtherCFDs/Dxtrade';
+
+const OtherCFDPlatformsList: FC = () => {
+    const { isFetching } = useAuthorize();
+    const { data, isFetchedAfterMount } = useDxtradeAccountsList();
+    const invalidate = useInvalidateQuery();
+    const hasDxtradeAccount = !!data?.length;
+
+    useEffect(() => {
+        if (!isFetching) {
+            invalidate('trading_platform_accounts');
+        }
+    }, [invalidate, isFetching]);
+
+    return (
+        <div className='pb-1200'>
+            <Text bold>Other CFD Platforms</Text>
+            <div className='grid grid-cols-3 gap-x-800 gap-y-2400 md:grid-cols-1 md:grid-rows-1'>
```

Comment:
better to use `lg` for the breakpoints currently we don't support `md` size screens